plugins/livelog/livelog.go

```diff
@@ -100,11 +103,42 @@ func (tp *taskPlugin) Finished(success bool) error {
 		return err
 	}
 	defer file.Close()
+
+	tempDir, err := runtime.NewTemporaryStorage(os.TempDir())
+	if err != nil {
+		return err
+	}
+
+	tempFile, err := tempDir.NewFile()
+	if err != nil {
+		return err
+	}
+
+	defer tempFile.Close()
```

Comment:
agreed, we should probably not even return an error, and instead teach the tempFile to log errors, because it should always work... anyways that's for another day.